Resources/Locale/en-US/communications/communications-console-component.ftl

```diff
@@ -23,5 +23,7 @@ comms-console-announcement-unknown-sender = Unknown
 # Comms console variant titles
 comms-console-announcement-title-station = Communications Console
 comms-console-announcement-title-centcom = Central Command
-comms-console-announcement-title-nukie = Syndicate Nuclear Operative
+# Delta-V:
+# comms-console-announcement-title-nukie = Syndicate Nuclear Operative
+comms-console-announcement-title-nukie = Unauthorized
```

Comment:
not a proper multiline comment + this should really go into its own file instead

Reply:
Could you clarify how you want this to look?